source/web/e2e/helpers.js

```diff
@@ -0,0 +1,155 @@
+    //   });
+    // }
+    // await fixtureServer.start();
+    // await fixtureServer.loadState(path.join(__dirname, 'fixtures', fixtures));
+    // if (dapp) {
+    //   let dappDirectory;
+    //   if (dappPath) {
+    //     dappDirectory = path.resolve(__dirname, dappPath);
+    //   } else {
+    //     dappDirectory = path.resolve(
+    //       __dirname,
+    //       '..',
+    //       '..',
+    //       'node_modules',
+    //       '@metamask',
+    //       'test-dapp',
+    //       'dist',
+    //     );
+    //   }
+    //   dappServer = createStaticServer(dappDirectory);
+    //   dappServer.listen(dappPort);
+    //   await new Promise((resolve, reject) => {
+    //     dappServer.on('listening', resolve);
+    //     dappServer.on('error', reject);
+    //   });
+    // }
+    // if (mockSegment) {
+    //   segmentStub = sinon.stub();
+    //   segmentServer = createSegmentServer((_request, response, events) => {
+    //     for (const event of events) {
+    //       segmentStub(event);
+    //     }
+    //     response.statusCode = 200;
+    //     response.end();
+    //   });
+    //   await segmentServer.start(9090);
+    // }
+    if (
+      process.env.SELENIUM_BROWSER === 'chrome' &&
+      process.env.CI === 'true'
+    ) {
+      // await ensureXServerIsRunning();
+    }
+    const { driver } = await buildWebDriver(driverOptions);
+    webDriver = driver;
+
+    await testSuite({
+      driver,
+      segmentStub,
+    });
+
+    if (process.env.SELENIUM_BROWSER === 'chrome') {
+      const errors = await driver.checkBrowserForConsoleErrors(driver);
+      if (errors.length) {
+        const errorReports = errors.map((err) => err.message);
+        const errorMessage = `Errors found in browser console:\n${errorReports.join(
+          '\n',
+        )}`;
+        if (failOnConsoleError) {
+          throw new Error(errorMessage);
+        } else {
+          console.error(new Error(errorMessage));
+        }
+      }
+    }
+  } catch (error) {
+    failed = true;
+    if (webDriver) {
+      try {
+        await webDriver.verboseReportOnFailure(title);
+      } catch (verboseReportError) {
+        console.error(verboseReportError);
+      }
+    }
+    throw error;
+  } finally {
+    if (!failed || process.env.E2E_LEAVE_RUNNING !== 'true') {
+      // await fixtureServer.stop();
+      // await ganacheServer.quit();
+      // if (ganacheOptions?.concurrent) {
+      //   await secondaryGanacheServer.quit();
+      // }
+      if (webDriver) {
+        await webDriver.quit();
+      }
+      // if (dappServer && dappServer.listening) {
+      //   await new Promise((resolve, reject) => {
+      //     dappServer.close((error) => {
+      //       if (error) {
+      //         return reject(error);
+      //       }
+      //       return resolve();
+      //     });
+      //   });
+      // }
+      // if (segmentServer) {
+      //   await segmentServer.stop();
+      // }
```

Comment:
I think the commented out code should probably be removed here until we actually implement those features. We can always reference the original source later if we need to.

Reply:
Removed